src/main/java/nextstep/ladder/ResultView.java

```diff
@@ -0,0 +1,45 @@
+package nextstep.ladder;
+
+import java.util.List;
+
+public class ResultView {
+    private final static String GAP = "     ";
+    private final static String LINE = "-----";
+    private final static String LADDER = "|";
+    private final static String LADDER_FORMAT = "%6s";
+
+
+    public static void printNames(List<String> names) {
+        names.stream().map(name -> String.format(LADDER_FORMAT, name)).forEach(System.out::print);
+        System.out.println();
+    }
+
+    public static void printLadderBoard(LadderBoard ladderBoard) {
+        ladderBoard.getLadderBoard().forEach(line -> {
+            printGap();
+            line.getPoints().forEach(point -> {
```

Comment:
view 입장에서 `Point`객체까지 알게하고 싶지 않네요 🤔 

```suggestion
            line.pointsToBoolean().forEach(point(Boolean 타입) -> {
```

가공해서 리턴해주는건 어떨까요?

- 규칙 9: 게터/세터/프로퍼티를 쓰지 않는다.

어쩌면 게터를 사용하지 말라는 많은 이유중 하나는, 
무조건 값을 내보내기 보다는 원하는 메시지를 받고 그에 대한 값을 패키징해서 보내주라는 것일수도 있겠네요 🤔